### PR TITLE
chore: add a note on resource management

### PR DESCRIPTION
**Pull request by darora**

### CODE REVIEW

**Review by olirice — APPROVED** [review]

File: web/docs/guides/database/resource-management.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+id: resource-management
+title: Resource Management
+description: Resource management and optimization
+---
+
+## Statement Timeouts
+
+By default, Supabase limits the maximum statement execution time to _3 seconds_ for users accessing the API using the anon key, and _8 seconds_ for authenticated users. This serves as a backstop against resource exhaustion due to either poorly written queries, or abusive usage.
```
Comment (steve-chavez) [inline web/docs/guides/database/resource-management.mdx:9]: I think it should mention the global timeout
```suggestion
By default, Supabase limits the maximum statement execution time to _3 seconds_ for users accessing the API using the anon key, and _8 seconds_ for authenticated users, all other users are limited by 2 minutes. This serves as a backstop against resource exhaustion due to either poorly written queries, or abusive usage.
```

Comment (darora) [reply web/docs/guides/database/resource-management.mdx]: Added a note

File: web/sidebars.js [hunk 2]
```diff
@@ -119,6 +119,7 @@ module.exports = {
             'guides/database/extensions/uuid-ossp',
           ],
         },
+        'guides/database/resource-management',
```
Comment (darora) [inline web/sidebars.js:122]: @burggraf / @kiwicopple - doesn't seem to be turning up on the preview at https://main-56wi8ry88-supabase.vercel.app/docs/guides ; any idea what I'm doing wrong?

Comment (soedirgo) [reply web/sidebars.js:122]: Shows up in https://docs-git-da-timeouts-supabase.vercel.app/docs/guides/database/resource-management

**Pull request merged**